src/applications/MachineShopSimulator.java

```diff
@@ -103,11 +83,23 @@ private void setUpJobs(SimulationSpecification specification) {
             theJob.putJobOnMachineQueue(this, firstMachine);
         }
     }
-
+    /**
+     * get task time for setting up jobs
+     * @param specification
+     * @param i
+     * @param j

```

Comment:
The params being named i and j is rather confusing; they could probably be renamed to jobNumber and taskNumber, and/or be explained in the signature

Reply:
That's a good point I didn't notice this.